### ISSUE TITLE
Uncaught SyntaxError: Unexpected token T

### ISSUE THREAD

**Issue by peterroelants** [autocomplete-python/autocomplete-python#187]
Steps to reproduce:
1. Create new Python File
2. Start typing (doesn't matter what)

**Atom Version**: 1.5.4
**System**: Ubuntu 14.04.3
**Thrown From**: [autocomplete-python](https://github.com/sadovnychyi/autocomplete-python) package, v1.7.0
### Stack Trace

Uncaught SyntaxError: Unexpected token T

```
At file:///usr/share/atom/resources/app.asar/static/index.html#%7B%22locationsToOpen%22%3A%5B%7B%22pathToOpen%22%3A%22%2Fhome%2Fpeter%2FWorkspace%2Fproduction-src%2Fdocument-check%22%2C%22initialLine%22%3Anull%2C%22initialColumn%22%3Anull%7D%5D%2C%22windowInitializationScript%22%3A%22%2Fusr%2Fshare%2Fatom%2Fresources%2Fapp.asar%2Fsrc%2Finitialize-application-window.js%22%2C%22resourcePath%22%3A%22%2Fusr%2Fshare%2Fatom%2Fresources%2Fapp.asar%22%2C%22devMode%22%3Afalse%2C%22safeMode%22%3Afalse%2C%22profileStartup%22%3Afalse%2C%22appVersion%22%3A%221.5.4%22%2C%22atomHome%22%3A%22%2Fhome%2Fpeter%2F.atom%22%2C%22shellLoadTime%22%3A424%2C%22initialPaths%22%3A%5B%22%2Fhome%2Fpeter%2FWorkspace%2Fproduction-src%2Fdocument-check%22%5D%7D:1

SyntaxError: Unexpected token T
    at Object.parse (native)
    at [object Object].module.exports._deserialize (/home/peter/.atom/packages/autocomplete-python/lib/provider.coffee:253:23)
    at provider.BufferedProcess.stdout (/home/peter/.atom/packages/autocomplete-python/lib/provider.coffee:51:10)
    at Socket.<anonymous> (/usr/share/atom/resources/app.asar/src/buffered-process.js:85:13)
    at emitOne (events.js:77:13)
    at Socket.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at Socket.Readable.push (_stream_readable.js:110:10)
    at Pipe.onread (net.js:523:20)
```
### Commands

```
     -2:01.3.0 core:copy (atom-text-editor.editor.is-focused)
     -2:00.4.0 core:paste (atom-text-editor.editor.is-focused)
     -1:58.8.0 core:save (atom-text-editor.editor.is-focused)
     -1:38.5.0 editor:newline (atom-text-editor.editor.is-focused)
  4x -1:37 core:backspace (atom-text-editor.editor.is-focused.autocomplete-active)
     -0:04.6.0 core:save (atom-text-editor.editor.is-focused)
  2x -0:04.0 editor:newline (atom-text-editor.editor.is-focused)
     -0:02.7.0 core:move-up (atom-text-editor.editor.is-focused)
     -0:02.1.0 snippets:next-tab-stop (atom-text-editor.editor.is-focused)
     -0:02.1.0 snippets:expand (atom-text-editor.editor.is-focused)
     -0:02.1.0 editor:indent (atom-text-editor.editor.is-focused)
```
### Config

``` json
{
  "core": {
    "themes": [
      "atom-dark-ui",
      "solarized-dark-syntax"
    ]
  },
  "autocomplete-python": {}
}
```
### Installed Packages

``` coffee
# User
Sublime-Style-Column-Selection, v1.5.1
autocomplete-paths, v1.0.2
autocomplete-python, v1.7.0
file-icons, v1.6.18
highlight-selected, v0.11.2
json-colorer, v0.3.0
json-schema, v0.1.15
language-docker, v1.1.6
language-lua, v0.9.4
linter, v1.11.3
linter-lua, v1.0.1
linter-pep8, v1.2.1
linter-pyflakes, v0.3.0
minimap, v4.20.0
minimap-highlight-selected, v4.4.0
pretty-json, v0.4.1
python-tools, v0.6.7

# Dev
No dev packages
```

**Comment by peterroelants** [comment]
Uninstalling and re-installing the "autocomplete-python" package seems to have fixed the issue for now.

**Comment by peterroelants** [comment]
Forget about my last comment, re-installing the package only solved the issue for 1 minute or so, as soon as I started editing some real code the issue returned.

**Comment by sadovnychyi** [comment]
Related to #140

**Comment by sadovnychyi** [comment]
Should be fixed in 1.8.9.
